specs/testdata/number.txt

```diff
@@ -0,0 +1 @@
+12345
```

Comment:
Can we try this with an accountID that begins with a `0`? It is a common problem that tools mishandle accountIDs that begin with 0s... I think it was EKS when it was first released the first 6 hours you couldn't use it any accounts that begin with a `0`

Reply:
Changed the case, but also added a test where we specifically assert that the name is preserved correctly (otherwise maybe it could be a string, but with the first 0 trimmed off) :)